mono/profiler/mono-profiler-log.c

```diff
@@ -4791,217 +4578,52 @@ mono_profiler_startup_log (const char *desc)
 	mono_profiler_startup (desc);
 }
 
+static ProfilerConfig config;
```

Comment:
It would be nice to move this near the top of the file and use it instead of all the global variables we have right now. Not a blocker though.